tests/target/configs-where_single_line.rs

```diff
@@ -1,3 +1,12 @@
+// rustfmt-where_single_line: true
+// Where style
+
+
+fn lorem_two_items<Ipsum, Dolor, Sit, Amet>() -> T 
+where Ipsum: Eq, Lorem: Eq {
```

Comment:
This should have regular where clause formatting, not single-lined (where_single_line should only apply if there is a single item in the clause).